storage/ndb/src/kernel/blocks/dbtup/DbtupTrigger.cpp

```diff
@@ -1805,7 +1812,27 @@ bool Dbtup::readTriggerInfo(TupTriggerData *const trigPtr,
         get_copy_tuple(&req_struct->prevOpPtr.p->m_copy_tuple_location);
   }
 
-  if (regTabPtr->need_expand(disk)) prepare_read(req_struct, regTabPtr, disk);
```

Comment:
After considering it in a deep way for a few hours I think I can conclude that this is never needed.
The code here is about reading the AFTER value, for DELETEs it only reads the PK, so no need
to worry about disk pages in this case. If an extra disk page exists, then it means that the transaction
has updated the disk part. This means that the AFTER value for the disk part is in the copy row.
Thus we will not make use m_disk_page_ptr in this case.